src/lcm/solution/solve_brute.py

```diff
@@ -75,31 +78,31 @@ def solve(
 
 def solve_continuous_problem(
     state_choice_space: StateChoiceSpace,
-    compute_ccv,
-    continuous_choice_grids,
-    vf_arr,
-    params,
-):
+    compute_ccv: Callable[..., Array],
+    continuous_choice_grids: dict[str, Array],
+    vf_arr: Array | None,
+    params: ParamsDict,
+) -> Array:
     """Solve the agent's continuous choices problem problem.
 
     Args:
-        state_choice_space (Space): Class with model variables.
-        compute_ccv (callable): Function that returns the conditional continuation
+        state_choice_space: Class with model variables.
+        compute_ccv: Function that returns the conditional continuation
             values for a given combination of states and discrete choices. The function
             depends on:
             - discrete and continuous state variables
             - discrete and continuous choice variables
             - vf_arr
             - params
-        continuous_choice_grids (list): List of dicts with 1d grids for continuous
+        continuous_choice_grids: List of dicts with 1d grids for continuous
             choice variables.
-        vf_arr (jax.Array): Value function array.
-        params (dict): Dict of model parameters.
+        vf_arr: Value function array.
+        params: Dict of model parameters.
 
     Returns:
-        jnp.ndarray: Jax array with continuation values for each combination of a
-            state and a discrete choice. The number and order of dimensions is defined
-            by the `gridmap` function.
+        Jax array with continuation values for each combination of a state and a
+            discrete choice. The number and order of dimensions is defined by the
```

Comment:
```suggestion
            set of all discrete choices. The number and order of dimensions is defined by the
```
?

Not perfect, but "a discrete choice" makes me think we are just looking at one of the potentially many discrete choices.